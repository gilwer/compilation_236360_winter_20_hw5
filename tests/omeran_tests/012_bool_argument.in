// use bool as func arg

void takesTwoBool(bool x, bool y) {
    if (x or y) {
      print("at least one is true!");
    }
}

void main() {
    takesTwoBool(3<4 and 4<5, true); // at least one is true!
}