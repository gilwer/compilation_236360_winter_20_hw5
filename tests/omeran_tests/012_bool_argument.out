at least one is true!
